code/datums/supplypacks.dm

```diff
@@ -608,7 +608,7 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 					/obj/item/clothing/suit/russofurcoat)
 	cost = 50
 	containertype = /obj/structure/closet/
-	containername = "Russian Clothing"
+	containername = "russian clothing crate"
```

Comment:
In English, adjectives derived from proper nouns remain capitalized. `russian` should be capitalized.